versions/ufo3/glyphs/glif.md

```diff
@@ -167,6 +167,18 @@ This element may occur any number of times.
 
 #### This element has no child elements.
 
+#### Anchor naming conventions
+
+The following are agreed upon standard naming conventions for the `name` of anchors for special uses.
+
+##### Ligature hcaret and vcaret
+
+Anchors with a `name` starting with `hcaret` or `vcaret` indicate horizontal or vertical position for ligature glyphs, for example to complie the [OpenType GDEF Ligature Caret List table]. Anchors with the name `hcaret` provide the `x` value for a ligature caret, and the `y` value is ignored. Likewise, anchors with the name `ycaret` provide the `y` values for a ligature caret, and the `x` value is ignored.
+
+Both names may be followed by an underscore and arbitrary text (`_X`) for designers or applications to have unique caret position names.
+
+`hcaret` and `vcaret` must not both occur in a glyph. There must be at least `n`-1 caret postions for `n` ligature components.
```

Comment:
- hcaret -> caret
- also add underscores here
- "at least" -> "exactly"
- minor: I would perhaps not use backticks around "n" here, but simply use unformatted cap N: "There must be exactly N-1 caret postions for N ligature components."

Reply:
What if someone wanted to skip over a glyph in a ligature? I don't think we should say exactly, but "no more than".